### PR TITLE
Add integration test setup & Tests for rlay-seed

### LINKED ISSUE #10: Set up Travis

### PR DESCRIPTION
**Pull request by hobofan**
- [x] Add method to use for resolving seeding reference tree
- [x] Order entities by number of transitive dependencies for easier / more correct seeding

CLOSES #10

### CODE REVIEW

**Comment by hobofan** [comment]
The original planned refactor of rlay-seed is not quite done and will be finished in a separate PR. Contents of the PR:

- Integration test setup against rlay-client & Neo4J via docker-compose
- Unit tests around seeding
- Integration test for rlay-seed
- Some seeding related functions for resolving reference names, which are not being used in rlay-seed yet

@MichaelHirn Ready for review

File: test/integration/docker.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  });
+  const waitForNeo4j = waitPort({
+    host: "127.0.0.1",
+    port: 7474,
+    output: "silent"
+  });
+  return Promise.all([waitForRlayClient, waitForNeo4j])
+    .then(() => {
+      // even after the ports are available the nodes need a bit of time to get online
+      return asyncSleep(30000);
```
Comment (MichaelHirn) [inline test/integration/docker.js]: This and line `41` above - did T CI introduce a solution for this since that file was first written?

**Review by MichaelHirn — COMMENTED** [review]
LGTM! 👍 

Just wondering about those hard 30s sleeps.

File: test/integration/docker.js (hunk 1 above)
Comment (hobofan) [reply test/integration/docker.js]: I turned it down to 5 secs, but I have a feeling that it's being ignored anyway 🤷‍♂

**Pull request merged**